WordPress/src/main/java/org/wordpress/android/ui/reader/actions/ReaderPostActions.java

```diff
@@ -470,4 +471,27 @@ private static String getEndpointForTag(ReaderTag tag) {
         return String.format("/read/tags/%s/posts", ReaderUtils.sanitizeWithDashes(tag.getTagName()));
     }
 
+    /*
+     * returns the passed endpoint without the unnecessary path - this is
+     * needed because as of 20-Feb-2015 the /read/menu/ call returns the
+     * full path but we don't want to use the full path since it may change
+     * between API versions (as it did when we moved from v1 to v1.1)
+     *
+     * ex: https://public-api.wordpress.com/rest/v1/read/tags/fitness/posts
+     *     becomes just                            /read/tags/fitness/posts
+     */
+    private static String getRelativeEndpoint(final String endpoint) {
+        if (endpoint != null && endpoint.startsWith("http")) {
+            int pos = endpoint.indexOf("/read/");
+            if (pos > -1) {
+                return endpoint.substring(pos, endpoint.length());
```

Comment:
Sorry, I had read b6bd5ba in the opposite way.
Wonder why we're adding the leading slash when both V1 and V1.1 of the REST Endpoints already have it in the URL
```
protected static final String REST_API_ENDPOINT_URL_V1 = "https://public-api.wordpress.com/rest/v1/";
protected static final String REST_API_ENDPOINT_URL_V1_1 = "https://public-api.wordpress.com/rest/v1.1/";
```
and remove our first leading slash internally by calling ```getAbsoluteURL ``` https://github.com/Automattic/android-wordpress-com-rest/blob/master/WordPressComRest/src/main/java/com/wordpress/rest/RestClient.java#L69